apps/frontend/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,31 @@
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	className?: string | undefined;
+	icon: JSX.Element;
+	label: string;
+	onClick: () => void;
+};
+
+const IconButton = ({
+	className = "",
```

Comment:
We don't want to pass className to common components, in case we need something, we will pass it a specific prop
```suggestion
```

Reply:
Noted :notebook:

Reply:
I just thought that maybe we needed the option to change the button background and so on